prisma/seed/user-seed.ts

```diff
@@ -0,0 +1,545 @@
+import { Prisma, PrismaClient } from "@prisma/client";
+const prisma = new PrismaClient();
+
+export async function executeUserSeed() {
+  await prisma.user.deleteMany({});
+
+  const roleAdmin = await prisma.role.findFirstOrThrow({
+    where: { name: "Admin" },
+  });
+
+  const roleMaintenanceTechnician = await prisma.role.findFirstOrThrow({
+    where: { name: "Mainentance Technician" },
+  });
+
+  const rolePassenger = await prisma.role.findFirstOrThrow({
+    where: { name: "Passenger" },
+  });
+
+  const roleTrainOperator = await prisma.role.findFirstOrThrow({
+    where: { name: "Train Operator" },
+  });
+
+  const roleTicketBoothAttendant = await prisma.role.findFirstOrThrow({
+    where: { name: "Ticket Booth Attendant" },
+  });
```

Comment:
Creio que o seed do role tem que ser separado do user.
O ORM executa sequencialmente cada seed baseado no timestamp de sua criação. Tem algum motivo especifico do porque você colocou esse techo de código aqui no user-seed.ts?

Reply:
Seria para pegar o uuid da Role de maneira dinamica toda vez que for rodar os seeds, mas deixei os uuids das Roles fixos mesmo. Vou criar um PR com essas alterações e remover esse trecho de código